source/develop/developer-guide/engine/engine-debug-obfuscated-ui.html.md

```diff
@@ -29,35 +29,36 @@ Why is this happening now? Recently we introduced some code that captures all ex
 
 #### Getting the full stack trace from the console
 
-As before the full stack trace is being written to the console, but the user is now aware of it via the pop-up message. To get the full stack trace you can open up the browser console. On most browsers you can start the development tools by pressing:
-
-      ctrl-shift-i
+As before the full stack trace is being written to the console, but the user is now aware of it via the pop-up message. To get the full stack trace you can open up the browser console. On most browsers you can start the development tools by pressing <kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>i</kbd>.
 
 Once the development tools are open, you can switch to the console tab. Scroll down to where you see the exception as shown in the following diagram:
-![]( exception_console.png "fig: exception_console.png")
+
+![](exception_console.png "fig: exception_console.png")
+
 Your exception will be different but the top line as highlighted will be the important one.
 
 #### Making sure we have the right symbol maps
 
 Now that we have the stack trace and the most likely culprit of the exception, we don't know what it is. When the UI is built, goes through a process of optimizing and obfuscating the original code. This is done to reduce the download size and optimize the performance of the application. The down side of this is that one can't easily tell what is going on. During this process another file is generated called a symbol map. This file maps the obfuscated code to the original source code. Due to the size of the file this is not installed by default. To install the symbol maps run the following command as root:
 
-      yum install ovirt-engine-webadmin-portal-debuginfo ovirt-engine-userportal-debuginfo
-      If you are running rhev
-      yum install rhevm-webadmin-portal-debuginfo rhevm-userportal-debuginfo
+    yum install ovirt-engine-webadmin-portal-debuginfo ovirt-engine-userportal-debuginfo
+    # If you are running rhev
+    yum install rhevm-webadmin-portal-debuginfo rhevm-userportal-debuginfo
```

Comment:
I just fixed the formatting in the original page, didn't delve into the content.  

But you are right, I'll add another patch to state it.